test/src/tests.js

```diff
@@ -636,6 +709,53 @@ describe('Rokt Forwarder', () => {
                     },
                 });
             });
+
+            it('should collect local session attributes and send to launcher.selectPlacements', async () => {
+                window.mParticle.Rokt.store.localSessionAttributes = {
+                    'custom-local-attribute': true,
+                    'secondary-local-attribute': true,
+                };
+
+                await window.mParticle.forwarder.init(
+                    {
+                        accountId: '123456',
+                        placementEventMapping: JSON.stringify([
+                            {
+                                jsmap: '-1484452948',
+                                map: '-5208850776883573773',
+                                maptype: 'EventClass.Id',
+                                value: 'foo-mapped-flag',
+                            },
+                            {
+                                jsmap: '1838502119',
+                                map: '1324617889422969328',
+                                maptype: 'EventClass.Id',
+                                value: 'ad_viewed_test',
+                            },
```

Comment:
Is having an actual mapping necessary here if we already have items in `localSessionAttributes`.  And if it's for illustrative purposes, then i think you should have the `value`s in the mapping match the keys for what is in `localSessionAttributes`